File: monte-carlo-simulation-matlab/monte_carlo.m
function [L,M] = monte_carlo(sample, k, nOfSamples)
    L = zeros(1, length(k));
    M = zeros(1, length(k));
   
    for i = 1:length(k)
        for n = 1:nOfSamples
            [ell,emm] = sample(k(i));
            L(i) = L(i)+ ell;
            M(i) = M(i)+emm;
        end

        L(i) = L(i)/nOfSamples;
        M(i) = M(i)/nOfSamples;  
    end
   
end

File: MCsim-Mod1/MC-sim-julia-Mod1/monte_carlo.jl

function monte_carlo(sample, parameters, nOfSamples)
    L = zeros(Float64, length(parameters))
    M = zeros(Float64, length(parameters))

    for i in 1:length(parameters)
        for n = 1:nOfSamples
            (ell :: Float64, emm :: Float64) = sample(parameters[i])
            L[i] +=  ell
            M[i] +=  emm
        end

        L[i] /=  nOfSamples
        M[i] /=  nOfSamples
    end

    return (L, M)
end
